test/e2e/tests/single_zone_e2e_test.go

```diff
@@ -1549,6 +1560,84 @@ var _ = Describe("GCE PD CSI Driver", func() {
 		Entry("with missing multi-zone label", multiZoneTestConfig{diskType: standardDiskType, readOnly: true, hasMultiZoneLabel: false, wantErrSubstring: "points to disk that is missing label \"goog-gke-multi-zone\""}),
 		Entry("with unsupported disk-type pd-extreme", multiZoneTestConfig{diskType: extremeDiskType, readOnly: true, hasMultiZoneLabel: true, wantErrSubstring: "points to disk with unsupported disk type"}),
 	)
+
+	DescribeTable("Should update metadata when providing valid metadata",
```

Comment:
If we cherry pick PRs back to previous release branches (before your change), is there a way to skip these test cases ? Otherwise I think these will fail against PRs merged to old release branches. In the test log, I see 
```
Ran 43 of 51 Specs in 1471.037 seconds
SUCCESS! -- 43 Passed | 0 Failed | 2 Pending | 6 Skipped
```

So I think there is a way to skip somewhere

Reply:
I changed the table to be pending, so this should skip for older versions. Is this sufficient?

Reply:
How does that make it skip for older versions? I see your tests now are pending even for this PR. I'm assuming that means they didn't run at all. What are the tests we are skipping now, and can we make those be skipped based on release branch?

Reply:
The other tests that were skipped are the tests in multi_zone_e2e_test.go. They get skipped if an environment variable isn't set. It doesn't seem like doing the same for these tests makes sense as they just wouldn't run on the tests for this PR. I set the tests to read the latest tag information and base the test skip on if the version is before v1.15.0.

Reply:
Actually, based on a past new feature / new e2e test (https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/commit/70148a9b6a0f55837a6b2c1e153c43976dc9b9e0#diff-2547c5a5595d73ed4720c414c96d15d919e53abf7441f91f1b2c7250196a2606) I don't think this is an issue I guess?

Reply:
So basing test skips off an environment variable is fine? I guess that would make sense especially given that the tests were verified to be passing from an earlier PR run